pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -125,48 +119,30 @@ export const KubernetesInventory: FC<KubernetesProps> = ({
         isVisible={deleteModalVisible}
         onClose={() => setDeleteModalVisible(false)}
       >
-        <Form
-          onSubmit={() => {}}
-          render={({ form, handleSubmit }) => (
-            <form onSubmit={handleSubmit}>
-              <>
-                <h4 className={styles.deleteModalContent}>
-                  {Messages.kubernetes.deleteModal.confirmMessage}
-                </h4>
-                <FormElement
-                  dataQa="form-field-force"
-                  label={Messages.kubernetes.deleteModal.labels.forceWrapper}
-                  element={
-                    <CheckboxField name="force" label={Messages.kubernetes.deleteModal.labels.force} />
-                  }
-                />
-                <HorizontalGroup justify="space-between" spacing="md">
-                  <Button
-                    variant="secondary"
-                    size="md"
-                    onClick={() => setDeleteModalVisible(false)}
-                    data-qa="cancel-delete-kubernetes-button"
-                  >
-                    {Messages.kubernetes.deleteModal.cancel}
-                  </Button>
-                  <Button
-                    variant="destructive"
-                    size="md"
-                    onClick={() => {
-                      if (kubernetesToDelete) {
-                        deleteKubernetes(kubernetesToDelete, Boolean(form.getState().values.force));
-                        setDeleteModalVisible(false);
-                      }
-                    }}
-                    data-qa="delete-kubernetes-button"
-                  >
-                    {Messages.kubernetes.deleteModal.confirm}
-                  </Button>
-                </HorizontalGroup>
-              </>
-            </form>
-          )}
-        />
+        <h4 className={styles.deleteModalContent}>{Messages.kubernetes.deleteModal.confirmMessage}</h4>
+        <HorizontalGroup justify="space-between" spacing="md">
+          <Button
+            variant="secondary"
+            size="md"
+            onClick={() => setDeleteModalVisible(false)}
+            data-qa="cancel-delete-kubernetes-button"
+          >
+            {Messages.kubernetes.deleteModal.cancel}
+          </Button>
+          <Button
+            variant="destructive"
+            size="md"
+            onClick={() => {
```

Comment:
Create a function for this and useCallback

Reply:
done